src/plugins/visualizations/public/components/visualization_container.tsx

```diff
@@ -17,13 +17,11 @@
  * under the License.
  */
 
-import { createMetricVisTypeDefinition } from './metric_vis_type';
-import { MetricVisComponent } from './components/metric_vis_component';
+import React from 'react';
 
-describe('metric_vis - createMetricVisTypeDefinition', () => {
-  it('has metric vis component set', () => {
-    const def = createMetricVisTypeDefinition();
+type Props = any;
```

Comment:
nit: Maybe `unknown` would be more appropriate here. Then you'd need to check if `className` and `children` exist in the `VisualizationContainer` component.

Reply:
added actual types